apps/admin/src/lib/auth/auth.middleware.ts

```diff
@@ -0,0 +1,24 @@
+import { cookies } from "next/headers";
+import "server-only";
+import { lucia } from "./auth.config";
+
+export const getMiddlewareSession = async () => {
```

Comment:
Muss ich das hier bei dem PR berücksichtigen, den ich später noch fürs Logging aufmache?

Reply:
Nein, hab das nur angepasst weil die getActionSession setzt Cookies, in der middleware wirft das einen Fehler.